lunar/distribution.yaml

```diff
@@ -59,6 +59,22 @@ repositories:
       url: https://github.com/ros/genmsg.git
       version: indigo-devel
     status: maintained
+  gennodejs:
+    doc:
+      type: git
+      url: https://github.com/RethinkRobotics-opensource/gennodejs.git
+      version: kinetic-devel
+    release:
+      tags:
+        release: release/lunar/{package}/{version}
+      url: https://github.com/RethinkRobotics-release/gennodejs-release.git
+      version: 2.0.1-0
+    source:
+      test_pull_requests: true
```

Comment:
Have you made sure the buildfarm has access to your repo / org unit (see http://wiki.ros.org/buildfarm/Pull%20request%20testing)?